README.md

```diff
@@ -1,8 +1,14 @@
-Webpack Strip
-=============
+# Webpack Strip
+
+[![npm version](https://badge.fury.io/js/webpack-strip.svg)](http://badge.fury.io/js/webpack-strip)
```

Comment:
First time I've seen us use this. Is this something we're going to put on other repos?